docs_user/assemblies/assembly_migrating-ceph-monitoring-stack.adoc

```diff
@@ -0,0 +1,39 @@
+[id="migrating-ceph-monitoring_{context}"]
+
+:context: migrating-ceph-monitoring
+
+= Migrating a {Ceph} cluster with monitoring stack components
+//kgilliga: SMEs, what do you think of the title? ^
```

Comment:
The title is good, I would only remove "with", because in this section you're going to literally migrate the `monitoring stack` component.

Reply:
If we only remove "with" the title doesn't really make sense. Maybe this title should be changed to "Migrating the monitoring stack component to RHOSP Controller nodes"?

Reply:
ok, but my point is we're migrating away from RHOSP controller nodes, so maybe "Migrating the monitoring stack component from RHOSP Controller nodes" ?

Reply:
We should focus on where the monitoring stack is being migrated *to*. So, what about: "Migrating the monitoring stack component to new nodes within an existing Red Hat Ceph Storage cluster"?

Apologies if I'm misunderstanding the intent.

Reply:
no problem at all and sorry, I think I just messed up trying to convey the intent of the chapter. I think it's fine, we can refine the title over the time if we need to make it shorter!